### PR TITLE
Fix some typos in the documentation

### PR DESCRIPTION
**Pull request by tbm**

### CODE REVIEW

File: doc/whitepaper/nightfall-v1.tex [hunk 1]
```diff
@@ -444,7 +444,7 @@ \subsection{ZoKrates JavaScript Wrapper}
       \texttt{./zkp/code/gm17/} & \texttt{.pcode} files\\
       \texttt{./zkp/code/README-tools-code-preprop.md} & explanation of \texttt{.pcode} syntax\\
       \texttt{./zkp/code/README-tools-trusted-setup.md} & how to automatically do the trusted setup\\
-      \url{https://github.com/EYBlockchain/zokrates-preprocessor} & how to manually transpile from \texttt{.pcode} to \texttt{.code}\\
+      \url{https://github.com/EYBlockchain/zokrates-preprocessor} & how to manually translate from \texttt{.pcode} to \texttt{.code}\\
```
Comment (aj-may) [inline doc/whitepaper/nightfall-v1.tex]: Can you please revert this one line.  I believe `transpile` was the intended word here.

https://stackoverflow.com/questions/44931479/compiling-vs-transpiling

File: doc/whitepaper/nightfall-v1.tex [hunk 2]
```diff
@@ -2886,7 +2886,7 @@ \subsection{Mint}
 \noindent
 Suppose Alice owns ERC-20 tokens of value $c$
 (denominated in the ERC-20 token's currency).
-Suppose Alice wishes to create a private token commitment, representating her ownership of value $c$.\\
+Suppose Alice wishes to create a private token commitment, represententing her ownership of value $c$.\\
```
Comment (skarred14) [inline doc/whitepaper/nightfall-v1.tex]: Guess you mean, `representing`

Comment (tbm) [reply doc/whitepaper/nightfall-v1.tex]: oops

File: doc/whitepaper/nightfall-v1.tex (hunk 1 above)
Comment (tbm) [reply doc/whitepaper/nightfall-v1.tex]: Ok, I'll put `transpile` back.  But I checked the docs for https://github.com/EYBlockchain/zokrates-preprocessor and it doesn't mention transpile anywhere, so I thought it was a typo. MAybe you can mention the word there as well.

**Comment by iAmMichaelConnor** [comment]
@tbm thanks very much for these changes. I’m going to push a restructured set of .tex files early next week (which will be a huge breaking change relative to the current whitepaper folder). I’ll include all of your typo corrections there.

@admins: DNM

**Comment by tbm** [comment]
@iAmMichaelConnor when you reorganize the .tex file, please make sure not to commit autogenerated cruft frorm LaTeX (as is currently done) -- add them to .gitignore.  (I was going to create a PR for this but won't bother based on your comments.)

**Comment by tbm** [comment]
Ok, I pushed a new patch that only fixes the types in the .md files. Maybe that can be applied.

**Comment by iAmMichaelConnor** [comment]
@tbm Thanks for the typo fixes to the readme instead. I was going to refer back to your typo fixes to the white paper's .tex file, but they seem to have been lost with your 'force-push' of the readme edits.

**Comment by iAmMichaelConnor** [comment]
@aj-may @skarred14 This PR has become just two typos, so I'm going to merge it.

**Comment by tbm** [comment]
@iAmMichaelConnor here are the typo fixes for the .tex file: https://github.com/tbm/nightfall/commit/703324a829d9f14a502c3c86f6d784d734ff900d

**Pull request merged**